website/content/en/status/report-2023-04-2023-06/mfsbsd-building-integration.adoc

```diff
@@ -0,0 +1,30 @@
+=== Integrate mfsBSD into the release building tools
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[Wiki Article] URL: link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[]
+
+Contact: Soobin Rho <soobinrho@FreeBSD.org>
+
+==== Purpose of this project
+
+This project creates an additional target of the weekly snapshots of -current and -stable versions of mfsBSD images in the src/release makefile.
+Currently, only the release versions of mfsBSD images are produced.
+This means they tend to get out of sync with the tools in base.
+This project aims to address that problem.
+
+==== What is mfsBSD?
+
+"mfsBSD is a toolset to create small-sized but full-featured mfsroot based distributions of FreeBSD that store all files in memory (MFS) [Memory File System] and load from hard drive, usb storage device or optical media.
+It can be used for a variety of purposes, including diskless systems, recovery partitions and remotely overwriting other operating systems."
+(link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[])
+
+Martin Matuska <mm@FreeBSD.org> is both the author of the white paper above and the maintainer of the mfsBSD repository: link:https://github.com/mmatuska/mfsbsd[]
+
+==== Location of this project
+
+This project is a GSoC 2023 (Google Summer of Code) project, under mentorship from Juraj Lutter <otis@FreeBSD.org> and Joseph Mingrone <jrm@FreeBSD.org>
+
+As a GSoC project, the official coding period is between May 29, 2023 and August 28, 2023.
+As a humble beginner in the open-source community, the author of this project welcomes all comments / suggestions / PR's to the project repository, which is where all the code for this project will be located for the duration of the official GSoC coding period: link:https://github.com/soobinrho/freebsd-src/tree/integrate-mfsBSD-building[]
```

Comment:
`link:https://github.com/soobinrho/freebsd-src/tree/integrate-mfs-building[project repository]`

Reply:
Awesome! I'll make corresponding changes when I get back home. Thank you for these suggestions; they are awesome changes for my status report!